dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -20,6 +20,21 @@ public sealed class SKContext
     /// <returns>Processed input, aka result</returns>
     public string Result => this.Variables.ToString();
 
+    /// <summary>
+    /// Whether all the context variables are trusted or not.
+    /// </summary>
+    public bool IsTrusted
+    {
+        get
+        {
+            return this.Variables.IsAllTrusted();
+        }
+        internal set
+        {
+            this.Variables.SensitiveInput = this.Variables.SensitiveInput.UpdateIsTrusted(value);
```

Comment:
flag all variables as not trusted (don't allow to change to True, only to False)